org.eclipse.paho.mqtt.c/src/Thread.c

```diff
@@ -216,21 +216,24 @@ sem_type Thread_create_sem()
     	else
     	{
     		int i;
-    		
+
     		named_semaphore_count++;
     		for (i = 0; i < MAX_NAMED_SEMAPHORES; ++i)
     		{
     			if (named_semaphores[i].name[0] == '\0')
-    			{ 
+    			{
     				named_semaphores[i].sem = sem;
-    				strcpy(named_semaphores[i].name, name);	
+    				strcpy(named_semaphores[i].name, name);
     				break;
     			}
     		}
     	}
 	#else
-		sem = malloc(sizeof(sem_t));
-		rc = sem_init(sem, 0, 0);
+        #if defined(__MACH__) || defined(__APPLE__)
+        #else
```

Comment:
how about using `#if !defined(__MACH__) && !defined(__APPLE__)` instead of an empty block?